client/rtcClient.js

```diff
@@ -63,21 +63,15 @@ function loginSuccess () {
       participants: $scope.roomUsers,
       meetings: $scope.roomName,
       meeting: $scope.roomName,
-      meetingUrl: location.href
-    })
-  }).catch(function (err) {
-    console.log('ERROR:', err)
-  }).then(function (result) {
-    console.log('meeting result:', result)
-    return app.service('participants').patch(easyrtc.myEasyrtcid, {
+      meetingUrl: location.href,
```

Comment:
Oh, it's not being patched to the participants object. All that code was removed!